modules/ROOT/pages/_partials/api-manager-usage.adoc

```diff
@@ -0,0 +1,17 @@
+= API Manager Usage Reports
+
+Usage reports show API Manager usage metrics on a monthly basis.
+
+== Access API Manager Usage Reports
+
+include::partial$usage-reports-tasks.adoc[tag=accessUsageReportsLegacy]
+
+. Click *API Manager*.
```

Comment:
Does this format correctly with the space between the steps in the partial?

Reply:
Yes. It looks fine in the build.